container_envvar.go

```diff
@@ -1,8 +1,46 @@
 package libyaml
 
+import "encoding/json"
+
 type ContainerEnvVar struct {
 	Name                  string `yaml:"name" json:"name"`
-	StaticVal             string `yaml:"static_val" json:"static_val"`
+	Value                 string `yaml:"value" json:"value"`
+	StaticVal             string `yaml:"static_val" json:"static_val"` // deprecated
```

Comment:
Tiny thing - I think putting `deprecated` above the field will make its (deprecated) usage show up different from non-deprecated fields in go editor tools

```go
    Value                 string `yaml:"value" json:"value"` 
    // deprecated: use Value instead
    StaticVal             string `yaml:"static_val" json:"static_val"`
```

Reply:
thanks!